docs/configuration/basics.md

```diff
@@ -102,12 +102,18 @@ the default system editor (Notepad on Windows and Nano on Unix systems) will be
 ### Customizing the editor
 
 If you want to use another editor, you can change the `EDITOR` (or `VISUAL`)
-environment variable to the path of your desired editor, such as:
+environment variable to the path of your desired editor.
 
+Linux/macOS example:
 ```
 EDITOR=/usr/bin/vim
 ```
 
+Windows example:
+
+![SystemPropertiesAdvanced_2024-05-16_02-08-15](https://github.com/espanso/website/assets/4579891/09e0c67e-a22e-4da0-a764-c9077db32aa9)
```

Comment:
Hi! Can I bother you asking where this link comes from? I don't see it on the `espanso/website` repo, maybe you missed adding this file?

Reply:
Hm I thought GitHub would handle this automatically so I just pasted the image into text editing form. Still opens on my side but I guess I have to put the image in the repo separately. Brb.

Reply:
Nvm I will cancel this PR and do this properly.

Reply:
There was no need! If you copy and paste the image inside of `/static/img/something.png` it was fine for me!

Reply:
GitHub's Web interface is PITA for such things so I just cloned the repo and did this locally. See https://github.com/espanso/website/pull/63.